packages/app/src/components/nominationBanner/nominationBanner.js

```diff
@@ -17,9 +18,16 @@ const NominationBanner = ({ nomination }) => {
   const minutes = newDate.toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' });
   const finalDate = `${time} – ${minutes}`;
   const [activeNomination, setActiveNomination] = useContext(ActiveNominationContext);
-  const declineStatus = 'Declined'
+  
+
+  const [modalState, setModalState] = useState(false);
+  const toggleModalState = () => {
+      setModalState(!modalState)
```

Comment:
Because React updates state asynchronously, there's a special way we should handle this kind of situation where we're updating the state based on the current version of the state. If we use `setModalState(currentState => !currentState)` instead of basing it off of `modalState` then the function inside `setModalState` will always have the most up-to-date version of the current state. With the way it's currently written, there's a chance it could use a stale value of the state and not update correctly.

Reply:
Done!